src/Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -82,6 +80,7 @@ public static class WebApiServiceCollectionExtensions
         /// Set to true if you want to debug, or just understand the JwtBearer events.
         /// </param>
         /// <returns></returns>
+        [Obsolete("rather use the override with the configSectionName, or the configure option actions")]
```

Comment:
Update the message and the one below to be consistent with what we have in the WebApp extensions class.

Reply:
I removed the obsolete methods.